.github/workflows/swift.yml

```diff
@@ -0,0 +1,99 @@
+name: Swift
```

Comment:
I would name this workflow with a more descriptive name. What does it really do?